integr8ly/cloud-resource-operator
# Add delete status

## Overview

Jira: https://issues.redhat.com/browse/INTLY-3779

*Note* : Included in this change, is some technical debt, updating and formatting import alias `croType`
## Verification

- Clone this branch
- Run `make cluster/prepare`
- Seed the cluster with both redis and postgres
- Run `make run`
- Delete both CR's while they are creating
- Ensure the status message reflects the current status of the resource and the current phase it is in
